### PR TITLE
new theme and updated color palettes

### PR DESCRIPTION
**Pull request by youngkidwarrior**
Writeup coming soon

### CODE REVIEW

**Comment by youngkidwarrior** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/452?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#452** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/452?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#442** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/442?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 1 other dependent PR ([#443](https://github.com/0xsend/sendapp/pull/443) <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/443?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* **#451** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/451?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `dev`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @youngkidwarrior and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/app/components/TopNav.tsx [hunk 1]
```diff
@@ -167,7 +167,7 @@ export function TopNav({
           <XStack>
             <IconSendLogo
               size={'$2.5'}
-              color={'$color12'}
+              color={'$color-1'}
```
Comment (0xBigBoss) [inline packages/app/components/TopNav.tsx:170]: this a mistake?

File: packages/app/components/sidebar/HomeSideBar.tsx [hunk 2]
```diff
@@ -30,22 +30,22 @@ import { ReferralLink } from '../ReferralLink'
 
 const links = [
   {
-    icon: <IconHome size={'$1.75'} />,
+    icon: <IconHome size={'$1.75'} color={'inherit'} />,
```
Comment (0xBigBoss) [inline packages/app/components/sidebar/HomeSideBar.tsx:33]: Does this work on native?

File: packages/app/features/auth/sign-in/screen.tsx [hunk 3]
```diff
@@ -85,32 +85,27 @@ const SignInScreensMobile = () => {
 const ContinueButton = ({ nextScreen }: { nextScreen: () => void }) => (
   <Stack w="100%" jc="center" py="$5" gap="$2">
     <Button
-      bc="transparent"
-      hoverStyle={{ backgroundColor: '$transparent', boc: '$accentBackground' }}
+      variant="outlined"
```
Comment (0xBigBoss) [inline packages/app/features/auth/sign-in/screen.tsx:88]: 🧙‍♂️

File: packages/ui/src/tamagui.config.ts [hunk 4]
```diff
@@ -8,6 +8,15 @@ import { radius } from './themes/token-radius'
 import { size } from './themes/token-size'
 import { space } from './themes/token-space'
 import { zIndex } from './themes/token-z-index'
+import { setupDev } from '@tamagui/core'
+
+setupDev({
```
Comment (0xBigBoss) [inline packages/ui/src/tamagui.config.ts:13]: Is it safe? Should we make it opt in somehow?

File: packages/ui/src/themes/masks.tsx
Comment (0xBigBoss) [inline packages/ui/src/themes/masks.tsx:1]: Mask on lfg nice work

File: packages/ui/src/themes/palettes.tsx [hunk 5]
```diff
@@ -2,64 +2,45 @@ import { objectFromEntries, objectKeys } from './helpers'
...
-    'hsl(0, 0%, 94.0%)',
-    'hsl(0, 0%, 92.0%)',
-    'hsl(0, 0%, 89.5%)',
-    'hsl(0, 0%, 81.0%)',
-    'hsl(0, 0%, 56.1%)',
-    'hsl(0, 0%, 50.3%)',
-    'hsl(0, 0%, 42.5%)',
-    lightColor,
-    darkTransparent,
+    '#FFFFFF',
```
Comment (0xBigBoss) [inline packages/ui/src/themes/palettes.tsx:15]: Interesting, there a reason we can’t import these as tokens instead of copy pasta?

**Review by 0xBigBoss — APPROVED** [review]
Amazing work brother. Send it.

File: packages/app/components/sidebar/HomeSideBar.tsx (hunk 2 above)
Comment (youngkidwarrior) [reply packages/app/components/sidebar/HomeSideBar.tsx:33]: Not sure. probably not. but we can cross that bridge when it comes

File: packages/ui/src/themes/palettes.tsx (hunk 5 above)
Comment (youngkidwarrior) [reply packages/ui/src/themes/palettes.tsx:15]: Let me see

File: packages/ui/src/tamagui.config.ts (hunk 4 above)
Comment (youngkidwarrior) [reply packages/ui/src/tamagui.config.ts:13]: It only runs when NODE_ENV=development

**Pull request merged**